storage/store_test.go

```diff
@@ -704,20 +643,12 @@ func TestStoreSendBadRange(t *testing.T) {
 // TODO(bdarnell): convert tests that use this function to use AdminSplit instead.
 func splitTestRange(store *Store, key, splitKey roachpb.RKey, t *testing.T) *Replica {
 	rng := store.LookupReplica(key, nil)
-	if rng == nil {
-		t.Fatalf("couldn't lookup range for key %q", key)
-	}
+	require.NotNil(t, rng, "couldn't lookup range for key %q", key)
 	desc, err := store.NewRangeDescriptor(splitKey, rng.Desc().EndKey, rng.Desc().Replicas)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 	newRng, err := NewReplica(desc, store)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if err = store.SplitRange(rng, newRng); err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
+	require.NoError(t, store.SplitRange(rng, newRng))
```

Comment:
In terms of readability I prefer the functions called on a separate line just like the case where they return more than an error.
err = store.SplitRange(rng, newRng)
require.NoError(t, err)